docker/gateway/Dockerfile-dev-noble

```
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM alpine:3.14 AS build-env
+LABEL maintainer="contact@graviteesource.com"
+
+ARG GRAVITEEAM_VERSION=0
+
+RUN apk add --update zip unzip
+
+ADD ./gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION}.zip /tmp/
+
+RUN unzip /tmp/gravitee-am-gateway-standalone-${GRAVITEEAM_VERSION}.zip -d /tmp/
+
+FROM eclipse-temurin:17.0.13_11-jre-noble
+LABEL maintainer="contact@graviteesource.com"
+
+ENV GRAVITEE_AM_HOME /opt/graviteeio-am-gateway
+
+COPY --from=build-env /tmp/gravitee-am-gateway-standalone-* /opt/graviteeio-am-gateway/
+
+RUN addgroup -gid 1001 graviteeio && \
```

```suggestion
RUN groupmod --new-name graviteeio ubuntu && \
```
To be aligned with alpine variant : group is 1000 and user is 1001. And  because group 1000 already exist, we can just rename it to have the same name everywhere.